settings.gradle

```diff
@@ -47,6 +47,7 @@ includeWithFlags ':javadoc',                             'java', 'publish', 'no_
 includeWithFlags ':benchmarks',              'java'
 includeWithFlags ':it:context-storage',      'java'
 includeWithFlags ':it:grpcweb',              'java', 'akka-grpc_2.13'
+includeWithFlags ':it:kotlin:error-message', 'java', 'relocate'
```

Comment:
How about renaming `:error-message` to `:annotated-service`? The `error-message` name is specific a bit for a module name and hard to predict what is tested in the module.

Reply:
OR just `:it:kotlin:`?
I think we don't want to create another module every time we add an IT test for kotlin.

Reply:
I changed to `:it:kotlin` for now.